### PR TITLE
(DB/Gameobject_addon) Fixed position for Ironforge Library plaques

### LINKED ISSUE #4627: Ironforge Library plaques relocated
Originally reported https://github.com/chromiecraft/chromiecraft/issues/100

##### ISSUE: FACTION SIDE:

Alliance

##### CONTENT PHASE:
 1-19

##### SMALL DESCRIPTION:

Zone: [Library (Ironforge)](https://wow.gamepedia.com/Library_(Ironforge))
Library's plaques are rotated and off place.
[Ironforge-Library-tablets](https://ibb.co/2NB3Sz9)

##### EXPECTED BLIZZLIKE BEHAVIOUR:
 

Plaques should be aligned to their basis.

##### CURRENT BEHAVIOUR:
 

Plaques are floating and not attached to their basis, making them invisible from a certain view.

##### STEPS TO REPRODUCE THE PROBLEM:

Step 1 - Venture into Ironforge's Library.
Step 2 - Take a look around.

##### AC HASH/COMMIT:

https://github.com/chromiecraft/azerothcore-wotlk/commit/a4115862de1cd6f37b70969c87df79f327457dd1

##### OPERATING SYSTEM:

Ubuntu 20.04

##### MODULES:

- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)

##### OTHER CUSTOMIZATIONS:

None.

##### SERVER:

ChromieCraft

<bountysource-plugin>

---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/96691444-ironforge-library-plaques-relocated?utm_campaign=plugin&utm_content=tracker%2F40032087&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F40032087&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by markvetx**
## Changes Proposed:
-  Added missing data to `gameobject_addon` table
-  Fixed positions for the plaques, now they are looking upwards

## Issues Addressed:
- Closes #4627 
- Closes https://github.com/chromiecraft/chromiecraft/issues/100

## How to Test the Changes:

1. `.go xyz -4605.23 -1242.93 503.38 0 3.80`
2. Run around the library and check the plaques

## Target Branch(es):
- [x] Master

 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1616258651093218025.sql [hunk 1]
```diff
@@ -0,0 +1,11 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1616258651093218025');
+
+DELETE FROM `gameobject_addon` WHERE `guid` IN (5141, 5205, 5382, 5398, 5405, 5425);
+
+INSERT INTO `gameobject_addon` VALUES (5141, 0, 0);
+INSERT INTO `gameobject_addon` VALUES (5205, 0, 0);
+INSERT INTO `gameobject_addon` VALUES (5382, 0, 0);
+INSERT INTO `gameobject_addon` VALUES (5398, 0, 0);
+INSERT INTO `gameobject_addon` VALUES (5405, 0, 0);
+INSERT INTO `gameobject_addon` VALUES (5425, 0, 0);
```
Comment (Si1ker) [inline data/sql/updates/pending_db_world/rev_1616258651093218025.sql:10]:
```suggestion
DELETE FROM `gameobject_addon` WHERE `guid` IN (5141, 5205, 5382, 5398, 5405, 5425);
INSERT INTO `gameobject_addon` (`guid`, `invisibilityType`, `invisibilityValue`) VALUES 
(5141, 0, 0),
(5205, 0, 0),
(5382, 0, 0),
(5398, 0, 0),
(5405, 0, 0),
(5425, 0, 0);


```

Comment (markvetx) [reply data/sql/updates/pending_db_world/rev_1616258651093218025.sql:10]: Whoops... My bad

**Review by Si1ker — APPROVED** [review]

**Comment by Si1ker** [comment]
@markvetx This one is wrong

image


### How it should be:
image
Source: https://wow.gamepedia.com/File:Highborne_Astrolabe.jpg

**Comment by markvetx** [comment]
@Si1ker Added one more object

**Comment by Si1ker** [comment]
Yep, looking good, good work!
image

**Comment by pklloveyou** [comment]
good job

**Pull request merged**